exercise_05.py

```diff
@@ -0,0 +1,15 @@
+def convert_millis(millis):
```

Comment:
It seems good at first glance, but I will provide a detailed check once you have solved the issues I am asking for.